src/OrbitGl/App.cpp

```diff
@@ -136,10 +136,12 @@ PresetLoadState GetPresetLoadStateForProcess(
 
 bool DoZoom = false;
 
-OrbitApp::OrbitApp(MainThreadExecutor* main_thread_executor,
+OrbitApp::OrbitApp(orbit_gl::MainWindowInterface* main_window,
+                   MainThreadExecutor* main_thread_executor,
                    const orbit_base::CrashHandler* crash_handler,
                    orbit_metrics_uploader::MetricsUploader* metrics_uploader)
-    : main_thread_executor_(main_thread_executor),
+    : main_window_{main_window},
```

Comment:
Maybe add a check that `main_window` is not `nullptr`?

Reply:
Done.